cli/src/cargo.rs

```diff
@@ -26,20 +27,25 @@ pub fn invoke(args: &Args) -> Result<()> {
         .arg("ts-rs/export")
         .arg("--features")
         .arg("ts-rs/generate-metadata")
-        .stdout(if args.no_capture {
+        .stdout(if cfg.no_capture {
             Stdio::inherit()
         } else {
             Stdio::piped()
         })
-        .env("TS_RS_EXPORT_DIR", path::absolute(&args.output_directory)?);
+        .env("TS_RS_EXPORT_DIR", path::absolute(&cfg.output_directory)?);
 
-    feature!(cargo_invocation, args, {
+    for (rust, ts) in &cfg.overrides {
+        let env = format!("TS_RS_INTERNAL_OVERRIDE_{rust}");
+        cargo_invocation.env(env, ts);
+    }
```

Comment:
the CLI reads the config file, and sets an environment variable for each override